src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -141,7 +142,12 @@ class MulticopterPositionControl : public control::SuperBlock, public ModulePara
 		(ParamInt<px4::params::MPC_POS_MODE>) MPC_POS_MODE,
 		(ParamInt<px4::params::MPC_ALT_MODE>) MPC_ALT_MODE,
 		(ParamFloat<px4::params::MPC_IDLE_TKO>) MPC_IDLE_TKO, /**< time constant for smooth takeoff ramp */
-		(ParamInt<px4::params::MPC_OBS_AVOID>) MPC_OBS_AVOID /**< enable obstacle avoidance */
+		(ParamInt<px4::params::MPC_OBS_AVOID>) MPC_OBS_AVOID, /**< enable obstacle avoidance */
+		(ParamBool<px4::params::MPC_WV_MAN_EN>) _wv_manual_enabled,
```

Comment:
Handle in the WV library (actual VTOL content in MPC minimal).

Reply:
Ok